tests/runtests.sh

```diff
@@ -180,6 +181,18 @@ if [ "" != "$SELENIUM_TIMEOUT" ]; then
     timeout_arg="--timeout=$SELENIUM_TIMEOUT"
 fi
 
+if [ "" != "$POKYDIR" ]; then
+    local_arg="--local"
+    poky_bind="-v $POKYDIR:/workdir/poky"
+fi
+
+    [[ -d $tempdir ]] || tempdir=$(mktemp -d --suffix toastertest --tmpdir)
```

Comment:
How would the "-d" ever be true? I don't see tempdir ever being set anywhere else.